styles.css

```diff
@@ -0,0 +1,252 @@
+* {
+  font-family: 'Open Sans', sans-serif;
+}
+
+header {
+  display: flex;
+  height: 15rem;
+  padding: 5% 3% 3% 8%;
+  border: none;
+  justify-content: space-between;
+}
+
+h1 {
+  font-family: 'Ibarra Real Nova', serif;
+  font-style: italic;
+  font-size: 1.5rem;
+  color: rgb(0, 179, 60);
+}
+
+h2 {
+  font-family: 'Ibarra Real Nova', serif;
+}
+
+h3 {
+  font-family: 'Ibarra Real Nova', serif;
+  font-size: 1.5rem;
+}
+
+h4 {
+  margin-left: 1rem;
+  margin-right: 1rem;
+}
+
+.container {
+  width: 80%;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.intro {
+  padding-right: 2rem;
+  font-size: 20px;
+}
+
+.card {
+  margin-left: 1rem;
+  margin-right: 1rem;
+}
+
+.hero-image {
+  background-image: url(images/header-image.jpg);
+}
+
+nav {
+  width: 100%;
+  background-color: rgb(240, 240, 240);
+  padding-left: 18rem;
+  margin-bottom: 2rem;
+  float: left;
+}
+
+.dog-card {
+  width: 20rem;
+  border: solid;
+  border-color: rgb(240, 240, 240);
+  margin-right: 1rem;
+  margin-bottom: 1rem;
+}
+
+.dog {
+  width: 100%;
+}
+
+.dog-button {
+  width: 100%;
+  background-color: rgb(0, 179, 60);
+  border: none;
+  align-items: center;
+  color: white;
+  cursor: pointer;
+}
+section {
```

Comment:
using so much padding A) isn't needed as the content should be left aligned and B) shouldn't be used on a generic selector like `section` instead you could create a class, and then use that class in sections as needed